gems/jquery-rails/CVE-2012-6708.yml

```diff
@@ -0,0 +1,46 @@
+---
+gem: jquery-rails
+cve: 2012-6708
+ghsa: 2pqj-h3vj-pqgw
+url: https://bugs.jquery.com/ticket/11290
+title: Cross-Site Scripting in jquery
+date: 2020-09-01
+description: |
+  Affected versions of `jquery` are vulnerable to cross-site scripting. This occurs because the main `jquery` function uses a regular expression to differentiate between HTML and selectors, but does not properly anchor the regular expression. The result is that `jquery` may interpret HTML as selectors when given certain inputs, allowing for client side code execution.
+
+  ## Proof of Concept
+  ```
+  $("#log").html(
+      $("element[attribute='<img src=\"x\" onerror=\"alert(1)\" />']").html()
+  );
+  ```
+
+  ## Recommendation
+
+  Update to version 1.9.0 or later.
```

Comment:
The description text is from the jQuery CVE. The version number should reflect `jquery-rails`, which has different version numbers than upstream jQuery.